mk/appveyor.bat

```diff
@@ -36,19 +36,21 @@ if %ERRORLEVEL% NEQ 0 (
   exit 1
 )
 
-set RUST_URL=https://static.rust-lang.org/dist/rust-%RUST%-%TARGET_ARCH%-pc-windows-msvc.msi
-echo Downloading %RUST_URL%...
 mkdir build
-powershell -Command "(New-Object Net.WebClient).DownloadFile('%RUST_URL%', 'build\rust-%RUST%-%TARGET_ARCH%-pc-windows-msvc.msi')"
+set RUSTUP_URL=https://win.rustup.rs/%TARGET_ARCH%
```

Comment:
Capitalize AppVeyor correctly in the commit message and add the "I agree to license my contributions to each file under the terms given at the top of each file I changed." to each commit.